src/main/java/com/github/vkorobkov/jfixtures/config/structure/tables/Tables.java

```diff
@@ -34,41 +34,41 @@ public CleanMethod getCleanMethod() {
     }
 
     public List<String> getBeforeInserts() {
-        return readArrayRecursively("before_inserts");
+        return readArray("before_inserts");
     }
 
     public List<String> getBeforeCleanup() {
-        return readArrayRecursively("before_cleanup");
+        return readArray("before_cleanup");
     }
 
     public List<String> getAfterInserts() {
-        return readArrayRecursively("after_inserts");
+        return readArray("after_inserts");
     }
 
     public Map<String, Object> getDefaultColumns() {
         return this.<Map<String, Object>>readProperty(
                 MapMerger::merge, "default_columns").orElse(Collections.emptyMap());
     }
 
-    private List<String> readArrayRecursively(String... sections) {
-        List<String> instructions = new ArrayList<>();
-        CollectionUtil.flattenRecursively(
-            readProperty(sections).orElse(Collections.emptyList()),
-            element -> instructions.add(String.valueOf(element))
-        );
-        return instructions;
+    private List readArray(String...sections) {
+        List result = new ArrayList();
+        readSections(sections).forEach(elem -> CollectionUtil.flattenRecursively(elem, result::add));
+        return result;
     }
 
-    private <T> Optional<T> readProperty(String ... sections) {
+    private <T> Optional<T> readProperty(String...sections) {
```

Comment:
I guess should we should have a space after String...

Reply:
I found 3 different styles of varargs in the project:
```java
1. (String...sections)

2. (String... sections)

3. (String ... sections)

4. (String ...sections)
```

Which one should we follow, @rodionovsasha ? I don't really have an opinion here

Reply:
I vote for (String... sections)
and intellij idea as well

Reply:
Makes perfect sense! 🍰